src/controller/genericmidi.cxx

```diff
@@ -623,28 +398,30 @@ void GenericMIDI::launchScene( int scene )
     
     if ( b->action == GRID_LAUNCH_SCENE )
     {
-      for( int i = 0; i < 5; i++ )
```

Comment:
Hi @georgkrause, I've looked at this patch change again - and I think it breaks the mapping. 1) Assume there is not a binding for every square / scene
2) Assume scene 2 is playing, and this event launches scene 4
3) Clip 2 needs to be written "off" and clip 4 needs a message for "on"

My understanding is that removing the loop causes the off-messages to not be written to all squares on the controller? If you prefer I can post a branch with this patch ported to latest git-head, and we can discuss on a new PR?